marlowe-dashboard-client/src/Template/State.purs

```diff
@@ -58,6 +58,16 @@ handleAction ::
   MonadAff m =>
   MonadAsk Env m =>
   Action -> HalogenM State Action ChildSlots Msg m Unit
+handleAction (SetTemplate contractTemplate) = pure unit -- handled in Play.State (see note [State])
+
+handleAction OpenTemplateLibraryCard = pure unit -- handled in Play.State (see note [State])
+
+handleAction (OpenCreateWalletCard tokenName) = pure unit -- handled in Play.State (see note [State])
+
+handleAction OpenSetupConfirmationCard = pure unit -- handled in Play.State (see note [State])
+
+handleAction CloseSetupConfirmationCard = pure unit -- handled in Play.State (see note [State])
```

Comment:
Super minor, but I've seen a lot of these `see note [State]` and I'm not sure which State to look for. Maybe we could say `see note [MainFrame.State]` or Play or the module the note is in.